card/mod/03_machines/lib/stylesheets/style_cards.scss

```diff
@@ -49,28 +49,92 @@ Choosing (b) will mean your CSS will not be affected by automated updates.
 /* -- card-headers and menus--*/
 
 .card-header {
+  position: relative;
   .card-header-title {
     display: inline-block;
     overflow: hidden;
+    span.glyphicon {
```

Comment:
OK to ignore this for now.  Fwiw, I was just thinking that if we added a class like "card-header-icon" or something, then we could have the margins work, so if some day someone adds a module to use font-awesome rather than glyphicon, this kind of margin styling would work.

Not worth sweating if you want to move on :)